caikit/runtime/interceptors/caikit_runtime_server_wrapper.py

```diff
@@ -42,30 +43,21 @@ class CaikitRuntimeServerWrapper(grpc.Server):
     instead.
     """
 
-    def __init__(
-        self, server, global_predict, intercepted_svc_descriptor, excluded_methods
-    ):
+    def __init__(self, server, global_predict, intercepted_svc_descriptor):
         """Initialize a new CaikitRuntimeServerWrapper
 
         Args:
             server(grpc.Server): The server that is being wrapped
             global_predict(function): A function that will accept an arbitrary
                 gRPC request message and a grpc.ServicerContext, and return
                 a suitable gRPC response message
-            excluded_methods(list(string)): An optional list of fully-qualified
```

Comment:
This removal is a bit hidden (i.e. the commit itself is updating test cases) - probably would be good to call out rationale in the PR